### PR TITLE
Use native Mkdocs page references

### PR DESCRIPTION
**Pull request by mikkeloscar**
Use the native mkdocs page references where the `.md` is included in the
link. This has the advantage of generating the correct links and you
don't have to think about adding `/` to the end of the link or not.
Fixes a few broken links.

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

File: docs/kubernetes/ingress-usage.md [hunk 1]
```diff
@@ -452,28 +452,28 @@ Set a Cookie in the response path of your clients.
...
 shows how to use filters for authorization.
 
 #### Basic Auth
 
     % htpasswd -nbm myName myPassword
 
-    basicAuth("/path/to/htpasswd")
-    basicAuth("/path/to/htpasswd", "My Website")
+    basicAuth("/pa../htpasswd")
+    basicAuth("/pa../htpasswd", "My Website")
```
Comment (szuecs) [inline docs/kubernetes/ingress-usage.md]: I think this is wrong

File: docs/kubernetes/ingress-usage.md [hunk 2]
```diff
@@ -79,9 +79,9 @@ Cloud migration.
 ### Multiple Ingresses defining the same route
 
 !!! Warning
-    
+
```
Comment (szuecs) [inline docs/kubernetes/ingress-usage.md]: Not sure if this will render correctly

File: docs/kubernetes/ingress-usage.md [hunk 3]
```diff
@@ -452,28 +452,28 @@ Set a Cookie in the response path of your clients.
...
 content of the token. Please check the filter documentation for that:
 
-- [oauthTokeninfoAnyScope](../../reference/filters/#oauthtokeninfoanyscope)
-- [oauthTokeninfoAllScope](../../reference/filters/#oauthtokeninfoallscope)
-- [oauthTokeninfoAnyKV](../../reference/filters/#oauthtokeninfoanykv)
-- [oauthTokeninfoAllKV](../../reference/filters/#oauthtokeninfoallkv)
+- [oauthTokeninfoAnyScope](../reference/filters.md#oauthtokeninfoanyscope)
+- [oauthTokeninfoAllScope](../reference/filters.md#oauthtokeninfoallscope)
+- [oauthTokeninfoAnyKV](../reference/filters.md#oauthtokeninfoanykv)
+- [oauthTokeninfoAllKV](../reference/filters.md#oauthtokeninfoallkv)
```
Comment (szuecs) [inline docs/kubernetes/ingress-usage.md]: Are you sure that removing ../ works here?

**Comment by szuecs** [comment]
@mikkeloscar did you checked https://opensource.zalando.com/skipper/tutorials/development/#user-documentation for the links changes, too?
One problem is that the root of the rendered page is at /skipper . I think you break all links with your pr.

File: docs/kubernetes/ingress-usage.md (hunk 2 above)
Comment (mikkeloscar) [reply docs/kubernetes/ingress-usage.md]: This works fine when I run it locally with `mkdocs serve -s`

image

File: docs/kubernetes/ingress-usage.md (hunk 1 above)
Comment (mikkeloscar) [reply docs/kubernetes/ingress-usage.md]: Ups, fixed

**Comment by mikkeloscar** [comment]
@szuecs thanks for the review, I was a bit too fast on the PR on Friday :/ Sorry about that!

I didn't think of the fact that the page is under `/skipper` I added the site URL to the `mkdocs.yaml` now, which should handle this. The `../` links works locally for me, but I'm not sure how to test this without merging?

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
@szuecs I pushed it and it seems to work fine, I can't find any links which doesn't work at least!

**Comment by mikkeloscar** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**